build_docs.pl

```diff
@@ -236,6 +239,9 @@ sub build_all {
             write_html_redirect( $build_dir->subdir( $_->{prefix} ),
                     $_->{redirect} );
         }
+
+        say "Writing nginx redirects";
+        write_nginx_redirects( $redirects, $build_dir, $temp_dir );
```

Comment:
What is your rule for whitespace inside function call parens?

Is it "extra space for dyadic and above function calls"?

Reply:
I has a somewhat shameful confession: I mostly don't care. Like, the only time I really want there to be some particular kind of whitespace is when I've got named parameters. I tend to make the whitespace look like the rest of the project around it and since this project sometimes uses `( $foo )` and sometimes uses `($foo)` I end up mixed up. Especially if I copying something. There is something to say for being consistent though. I think more of the project uses `( $foo )` I should probably stick with that.

Reply:
Thanks for replying. Not a huge deal to me either, to be frank, so sorry if it sounded picky.

Reply:
Note at all!